tests/pricing_models/test_pricing_model_utils.py

```diff
@@ -316,3 +538,11 @@ def test_calc_liquidity(self):
         """Execute the test"""
         self.run_calc_liquidity_test(sim_utils.get_pricing_model("hyperdrive"))
         self.run_calc_liquidity_test(sim_utils.get_pricing_model("yieldspace"))
+
+    def test_calc_bond_for_target_apr(self):
+        """Execute the test"""
+        self.run_calc_bond_for_target_apr(sim_utils.get_pricing_model("hyperdrive"))
```

Comment:
should this also have a yieldspace version?

Reply:
its pricing model agnostic, will leave a comment